### DISCUSSION TITLE
Is there API to disable Service Worker support in WebView2

### DISCUSSION THREAD

**Discussion by DexterWoo** [MicrosoftEdge/WebView2Feedback#3968] [Q&A]
Is there API to disable Service Worker support in WebView2? 
due the the service worker requests won't be hooked by the WebResourceRequested api, we'd like to try disable the service worker support and implement our own way of caching resources.

**Comment by vbryh-msft** [comment]
Hi @DexterWoo, the support for WebResourceRequested event for service workers will be available in the next prerelease SDK as experimental feature.
